src/main/java/org/fundacionjala/coding/manuel/Kata14.java

```diff
@@ -0,0 +1,28 @@
+package org.fundacionjala.coding.manuel;
+
+import java.util.Arrays;
+
+/**
+ * Created by Administrator on 3/16/2018.
+ */
+public class Kata14 {
+    //public static final int LENGTH_NUM = 1;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)